### ISSUE TITLE
In map search component, infobox should be focusable via TAB and placed before the coordinates text area

### ISSUE THREAD

**Issue by rscipien** [CeON/fairchive#2854]
It is in fact a WCAG issue:

* if we give users some instructions regarding what is expected in a given field, it should be given before the respective field, to give them chance to avoid creating invalid input,
* it should be possible to tab into the infobox to allow screen reader to read the content of the infobox (similarly to our question mark prompts – they are tabable as well and for the same reason).
